webpack.config.web.dev.babel.js

```diff
@@ -71,7 +71,7 @@ console.log('BLIP_URL =', blipUrl);
 
 const output = {
   path: path.join(__dirname, 'dist'),
-  publicPath: isDev ? `${devPublicPath}/` : '/',
+  publicPath: isDev ? `${devPublicPath}/` : '/uploader/',
```

Comment:
Should we remove the trailing slash here as well?

Reply:
I believe that `publicPath` is supposed to have a trailing slash.